params/gyroid_sides.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
consider having this fractal replace `gyroid`, this one turned out a lot better.